vm/interpreter/src/default_runtime.rs

```diff
@@ -238,15 +238,18 @@ where
         value: TokenAmount,
         params: Serialized,
     ) -> Result<Serialized, ActorError> {
-        let msg = UnsignedMessage::builder()
-            .from(from)
-            .to(to)
-            .method_num(method)
-            .value(value)
-            .params(params)
-            .gas_limit(self.gas_available())
-            .build()
-            .expect("Message creation fails");
+        let msg = UnsignedMessage {
+            from,
+            to,
+            method_num: method,
+            value,
+            params,
+            gas_limit: self.gas_available(),
+            version: Default::default(),
+            sequence: Default::default(),
+            gas_fee_cap: Default::default(),
+            gas_premium: Default::default(),
```

Comment:
Could probably get away with  ..Default::default() for all the fields that are automatic. But might be better for readability if everything was explicit.

Reply:
@austinabell and i talked about it. It's not possible to do that unless UnsignedMessage is Default

Reply:
Completely failed to notice it didn't implement Default

Reply:
it can't because there are addresses in the struct, which cannot implement default